cmd/lifecycle/analyzer.go

```diff
@@ -23,116 +23,95 @@ import (
 )
 
 type analyzeCmd struct {
-	//flags: inputs
-	analyzeArgs
-	stackPath string
-	uid, gid  int
-
-	//flags: paths to write data
-	analyzedPath string
+	analyzeInputsForCreator
+	additionalTags  cmd.StringSlice
+	analyzedPath    string
+	cacheImageRef   string
+	legacyCacheDir  string
+	legacyGroupPath string
+	outputImageRef  string
+	stackPath       string
+	uid, gid        int
 }
 
-type analyzeArgs struct {
-	cacheImageRef    string
+// Inputs needed when run by creator
+type analyzeInputsForCreator struct {
```

Comment:
I wonder if this rename is truly needed. It would be annoying to rename the struct for all the other phases. Maybe we just enforce the meaning with a comment (this is already mostly true for the other phases).

Reply:
Put back in d1988a83